avaframe/tests/test_getStats.py

```diff
@@ -22,27 +23,27 @@ def test_getStats(tmp_path):
 
     # define test directory
     avaName = 'avaStats'
-    avaDirtmp = os.path.join(tmp_path, avaName)
-    avaDirPeakFiles = os.path.join(tmp_path, avaName, 'Outputs', 'com1DFA', 'peakFiles')
-    avaDirConfigFiles = os.path.join(tmp_path, avaName, 'Outputs', 'com1DFA', 'configurationFiles')
+    avaDirtmp = pathlib.Path(tmp_path, avaName)
+    avaDirPeakFiles = avaDirtmp / 'Outputs' / 'com1DFA' / 'peakFiles'
+    avaDirConfigFiles = avaDirtmp / 'Outputs' / 'com1DFA' / 'configurationFiles'
     os.makedirs(avaDirtmp)
     os.makedirs(avaDirPeakFiles)
     os.makedirs(avaDirConfigFiles)
```

Comment:
Path.mkdir() should do the job, this way we can get rid of import os

Reply:
i make an issue for this